app/main.py

```diff
@@ -2,5 +2,26 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    result_dict = {}
+
+    def wrapper(*args, **kwargs) -> None:
```

Comment:
you can use only args - 

    def wrapper(*args) -> Callable:
        if args in caching_result:

Reply:
Ok